aws/table_aws_mgn_application.go

```diff
@@ -0,0 +1,202 @@
+)
+
+//// TABLE DEFINATION
+
+func tableAwsMGNApplication(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_mgn_application",
+		Description: "AWS MGN Application",
+		List: &plugin.ListConfig{
+			Hydrate: ListAwsMGNApplication,
+			IgnoreConfig: &plugin.IgnoreConfig{
+				// UninitializedAccountException - This error comes up when default template are not set for a particular region.
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"UninitializedAccountException"}),
+			},
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "application_id", Require: plugin.Optional},
+				{Name: "wave_id", Require: plugin.Optional},
+				{Name: "is_archived", Require: plugin.Optional, Operators: []string{"=", "<>"}},
+			},
+		},
+		GetMatrixItemFunc: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "Application name.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "application_id",
+				Description: "Application ID.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ApplicationID"),
+			},
+			{
+				Name:        "arn",
+				Description: "The Amazon Resource Name (ARN) of the application",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "creation_date_time",
+				Description: "Application creation dateTime.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "description",
+				Description: "Application description.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "is_archived",
+				Description: "Application archival status.",
+				Type:        proto.ColumnType_BOOL,
+			},
+			{
+				Name:        "last_modified_date_time",
+				Description: "Application last modified dateTime.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "wave_id",
+				Description: "Application wave ID.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("WaveID"),
+			},
+			{
+				Name:        "application_aggregated_status",
+				Description: "Application aggregated status.",
+				Type:        proto.ColumnType_JSON,
+			},
+
+			// Steampipe standard columns
+			{
+				Name:        "tags",
+				Description: "A list of tags attached to the Cluster.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Name"),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Arn").Transform(transform.EnsureStringArray),
+			},
+		}),
+	}
+}
+
+// // LIST FUNCTION
+func ListAwsMGNApplication(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
```

Comment:
```suggestion
func ListAwsMGNApplications(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
```

Reply:
Please update all instances of `ListAwsMGNApplication` to `ListAwsMGNApplications`